### PR TITLE
sbc, disable for broken gcc2 build

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Build broken at buildmaster: https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/5494/builds/82439.log

@korli this isn't used by anything I can find, should be OK to disable for gcc2 I guess?

File: media-libs/sbc/sbc-2.0.recipe [hunk 1]
```diff
@@ -12,12 +12,12 @@ COPYRIGHT="2004-2010 Marcel Holtmann
...
 	GNU LGPL v2.1"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://www.kernel.org/pub/linux/bluetooth/sbc-$portVersion.tar.gz"
 CHECKSUM_SHA256="897c1463fc6ad25a7822dfa2ce53c340c4ac8054eab381f72aa7d3bf16517738"
 PATCHES="sbc-$portVersion.patchset"
 
-ARCHITECTURES="all"
+ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```
Comment (augiedoggie) [inline media-libs/sbc/sbc-2.0.recipe:21]:
```suggestion
ARCHITECTURES="all !x86_gcc2"
SECONDARY_ARCHITECTURES="x86"
```
It looks like the `!x86_gcc2` was put on the wrong line to begin with and should already be disabled.

Comment (Begasus) [reply media-libs/sbc/sbc-2.0.recipe:21]: Thought it looked weird already :)

**Comment by korli** [comment]
> @korli this isn't used by anything I can find, should be OK to disable for gcc2 I guess?

Yes, it seems it was so since the "all" keyword introduction.

**Pull request merged**